diem-move/writeset-transaction-generator/templates/disable_parallel_execution.move

```diff
@@ -0,0 +1,6 @@
+script {
```

Comment:
I'm wondering if there's a need for the script functions defined in the Move configs commit if the plan is for these to be used (or if only those or these should exist). Are they only there for testing purposes?

Reply:
IMHO they are not only for testing purposes. When we want to enable/disable parallel execution in testing/production environment, we will use those scripts to generate the writeset payload needs to be sent to the network.